src/PracticeDeck.java

```diff
@@ -0,0 +1,111 @@
+import java.util.Collections;
+import java.util.ArrayList;
+
+// For setting seed: // https://stackoverflow.com/questions/6284589/setting-a-seed-to-shuffle-arraylist-in-java-deterministically
+import java.util.Random;
+
+public class PracticeDeck {
```

Comment:
this class feels a little weird - these methods feel like they should be part of a main driver class somewhere, is InterfaceHandler your main drvier class?

Reply:
maybe they should be part of the DeckHandler